.circleci/config.yml

```diff
@@ -175,7 +175,31 @@ jobs:
           root: _save
           paths:
           - images.tar
-      - run: make lint
+
+  shellcheck:
+    docker:
+      - image: koalaman/shellcheck-alpine:v0.6.0
+    steps:
+      - checkout
+      - run:
+          name: Check shell scripts
+          command: /root/project/bin/check_shell_scripts.sh
+
+  lint:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Run linters
+          command: make lint
+
+  test:
```

Comment:
IMO, don't add a new test job as the existing install-cni job is for performing `make test`.

Reply:
Done.